### ISSUE TITLE
Questions on reproducing results from the paper

### ISSUE THREAD

**Issue by haruishi43** [jianlong-yuan/semi-mmseg#6]
Thank you for open sourcing your paper! The results presented in the paper are extremely compelling.

I have some questions and requests:
1. do you plan to release all of the configs you used in the paper? I would greatly appreciate it so I can reproduce the Cityscapes results. Also, are there any weights that you can share to verify evaluations for Cityscapes and Pascal VOC?
2. do you have a specific way to evaluate the trained models? I was going to start training the only available config ('cps_meanteacher_3b_w1.5_...'), and was wondering if the saved checkpoints can be directly used for `tools/test.py`.
3. For `ExpSemiLossCPSFAWS3` (which I assume is the Lst and Lss losses) seem to not use knowledge selection. Reading the code, the `self.prob` seems to be the threshold, but I don't see anywhere where that is set to 0.95 as in the paper.

**Comment by Hugo-cell111** [comment]
Hi! Have you reproduced the group of experiment of cps_meanteacher_3b_w1.5_?

**Comment by haruishi43** [comment]
@Hugo-cell111 hi, and I haven't. I mainly do experiments on Cityscapes, so I haven't downloaded Pascal yet. Did you get the code to run?

**Comment by Hugo-cell111** [comment]
No, I haven't got the code yet. Could you successfully reproduce the results of Cityscapes?

**Comment by haruishi43** [comment]
No, I'm waiting for their response for the config they used for their cityscapes experiments.
